src/schemas/project.ts

```diff
@@ -0,0 +1,36 @@
+import gql from "graphql-tag";
+
```

Comment:
I didn't find anything on output types. Is that part of another ticket?

Reply:
going to work on that tomorrow along with the ret of the research outputs